tests/Dockerfile.nginx

```diff
@@ -0,0 +1,15 @@
+FROM nginx:1.13.3
+
+MAINTAINER Conjur Inc
+
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get update && \
+    apt-get install -y iputils-ping procps openssl
```

Comment:
Can you add `rm -rf /var/lib/apt/lists/*` at the end of this command? It makes the cached image smaller and more light-weight.